reference/latest/src/main/java/com/example/docs/item/ModItems.java

```diff
@@ -109,6 +111,19 @@ public class ModItems {
 	);
 	// :::2
 
+	/**Helper methods for registering items (Fellteros) <br>
+	 * Item registration would like this:
+	 * <blockquote><pre>
+	 *     public static final Item TEST = registerItem("test", Item::new)
+	 * </pre></blockquote>*/
+	private static Item registerItem(String name, Function<Item.Settings, Item> function) {
+		return Registry.register(Registries.ITEM, Identifier.of(FabricDocsReference.MOD_ID, name), function.apply(new Item.Settings().registryKey(keyOfItem(name))));
+	}
+	private static RegistryKey<Item> keyOfItem(String name) {
+		return RegistryKey.of(RegistryKeys.ITEM, Identifier.of(FabricDocsReference.MOD_ID, name));
+	}
```

Comment:
don't we already have helpers for registration that are used in most items? ^^'

Reply:
Yes, not sure why this was added. Just use the `register` we already have.